packages/cli/src/lib/utils.test.ts

```diff
@@ -73,25 +72,6 @@ describe('utils', () => {
   });
 
   describe('getAxeSource', () => {
-    describe('mock file', () => {
-      beforeEach(() => {
-        mock({
-          '/node_modules/axe-core': {},
-          '../node_modules/axe-core': {
-            'axe.js': mock.load(require.resolve('axe-core'))
-          }
-        });
-      });
-
-      afterEach(() => {
-        mock.restore();
-      });
-      it('fall back to use `locally` installed axe-core', () => {
```

Comment:
Can we implement a different test rather than removing this?

Reply:
The fact that it defaults to some locally installed axe-core is already tested indirectly through several other test cases. The node_module resolution code has been simplified to `axePath = require.resolve('axe-core/axe.js');`, which I don't really think requires dedicated testing.